packages/compass-components/package.json

```diff
@@ -34,48 +34,49 @@
     "@dnd-kit/core": "^6.0.7",
     "@dnd-kit/sortable": "^7.0.2",
     "@dnd-kit/utilities": "^3.2.1",
-    "@leafygreen-ui/badge": "^9.0.2",
-    "@leafygreen-ui/banner": "^9.0.2",
-    "@leafygreen-ui/button": "^22.0.2",
-    "@leafygreen-ui/card": "^12.0.2",
-    "@leafygreen-ui/checkbox": "^14.0.2",
-    "@leafygreen-ui/chip": "^3.0.12",
+    "@leafygreen-ui/badge": "^10.0.2",
+    "@leafygreen-ui/banner": "^10.0.2",
+    "@leafygreen-ui/button": "^24.0.2",
+    "@leafygreen-ui/card": "^13.0.2",
+    "@leafygreen-ui/checkbox": "^18.0.0",
+    "@leafygreen-ui/chip": "^4.0.2",
     "@leafygreen-ui/code": "^16.0.2",
```

Comment:
I'm guessing you're skipping this because of the failing postinstall, it's failing due to @lg-tools dependency not being included in the package deps. We can ask lg team if they intentionally added postinstall here and not something like prepublishOnly and as a workaround we can also just add @lg-tools/build package as a dev dependency to make sure all leafygreen is updated to latest, this is important